src/app/config/config.py

```diff
@@ -22,6 +22,12 @@ def __init__(self):
         self.PUBLISH_DATASET_ERROR_TOPIC_ID = get_value_from_env(
```

Comment:
I do not think we need the config values unless we wanted to create the infrastruture locally. i need to look into if I can write code to add a push subscription. The endpoint used can be used like a standard endpoint anyway.